lib/nnue_training_data_formats.h

```diff
@@ -7577,6 +7578,11 @@ namespace binpack
 
                     if (!m_localBuffer.empty())
                     {
+                        // now shuffle the local buffer
+			std::random_device rd;
+			static thread_local std::mt19937 g(rd());
```

Comment:
do static thread_local std::mt19937 g(std::random_device()); so that the random device it's created once